library/system.lisp

```diff
@@ -86,13 +86,14 @@ While the result will always contain microseconds, some implementations may retu
     (lisp UFix ()
       (cl:get-internal-real-time)))
 
-  #+sbcl
-  (declare get-bytes-consed (Unit -> UFix))
-  #+sbcl
+  (declare get-bytes-consed (Unit -> (Optional UFix)))
   (define (get-bytes-consed)
```

Comment:
maybe we should have this be `monotonic-bytes-consed` and the docs should be something like

> Return the number of bytes consed since some unspecified point in time. The difference between two successive calls to this function represents the number of bytes consed in that period of time.